circle.yml

```diff
@@ -0,0 +1,34 @@
+machine:
+  post:
+  - cd $HOME && git clone --depth 1 -v git@github.com:clever/ci-scripts.git && cd ci-scripts && git show --oneline -s
+  - $HOME/ci-scripts/circleci/golang-install 1.8
+  services:
+  - docker
+checkout:
+  post:
+  - $HOME/ci-scripts/circleci/golang-move-project
+compile:
+  override:
+  - make install_deps
+  - make build
+test:
+  override:
+  - make test
+  - $HOME/ci-scripts/circleci/report-card $RC_DOCKER_USER $RC_DOCKER_PASS "$RC_DOCKER_EMAIL" $RC_GITHUB_TOKEN
+deployment:
+  master:
+    owner: Clever
+    commands:
+    - $HOME/ci-scripts/circleci/docker-publish $DOCKER_USER $DOCKER_PASS "$DOCKER_EMAIL" $DOCKER_ORG
+    - $HOME/ci-scripts/circleci/catapult-publish $CATAPULT_URL $CATAPULT_USER $CATAPULT_PASS $APP_NAME
+    - $HOME/ci-scripts/circleci/dapple-deploy $DAPPLE_URL $DAPPLE_USER $DAPPLE_PASS $APP_NAME dev-infra confirm-then-deploy
+    - $HOME/ci-scripts/circleci/dapple-deploy $DAPPLE_URL $DAPPLE_USER $DAPPLE_PASS $APP_NAME production confirm-then-deploy
```

Comment:
Agreed with the issue that people might too quickly click "deploy" right now, though.

Since it's just me and I plan to do the rollout today, I'll go ahead and keep this.